### PR TITLE
Add type VenueRoom

### PR DESCRIPTION
**Pull request by brettski**
Adds rooms to venues stored as a sub-collection of the venue. 

VenueRoom id's are created from venueId and room name. e.g.(`OAdI0rVjE9D8ZoheAWMN_mangrove`)

### CODE REVIEW

File: src/graphql/typeDefs/mutations/venueRooms.graphql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type VenueRoomsMutation {
+  "Create a new Venue Room"
+  create(venueRoom: VenueRoomCreateInput!): VenueRoom! @auth(requires: "venues")
```
Comment (csell5) [inline src/graphql/typeDefs/mutations/venueRooms.graphql:3]: why not just admin? do we really need another role?

**Review by csell5 — CHANGES_REQUESTED** [review]

File: src/graphql/typeDefs/mutations/venueRooms.graphql (hunk 1 above)
Comment (brettski) [reply src/graphql/typeDefs/mutations/venueRooms.graphql:3]: The role was already there, I used the same role which was on Venue currently.

Comment (csell5) [reply src/graphql/typeDefs/mutations/venueRooms.graphql:3]: Did we ever really create it? Or better said now that we're here is it really needed any longer? Or is it just over completing things?

Comment (brettski) [reply src/graphql/typeDefs/mutations/venueRooms.graphql:3]: I see no use for it. My assumption is that we added it as others may make venue changes. I planned to use admin and when I saw that the other venue  mutations used, 'venue', I used it. It's a good opportunity to remove it. If we decide to, I'll remove it from auth0 too.

**Review by csell5 — APPROVED** [review]

**Pull request merged**